### ISSUE TITLE
Add an Apache libCloud compute module?

### ISSUE THREAD

**Issue by d33tah** [Arubacloud/pyArubaCloud#38]
Hi,

I just noticed that your library fits very well into the concept of "compute" module of Apache libCloud. Have a look at how many cloud companies they already support there:

https://libcloud.readthedocs.io/en/latest/supported_providers.html

Could you please add a way to use Aruba as well, at least for compute? Here's an example compute driver for LibCloud:

https://github.com/apache/libcloud/blob/trunk/libcloud/compute/drivers/vpsnet.py

**Comment by arubalucalasagni** [comment]
Hi d33tah,

Thank you for your request, we are evalueting to support an Apache libCloud Provider for Aruba and soon we could start to develop it.

**Comment by ZuluPro** [comment]
@d33tah Why create an libcloud module and not just make PR ?

**Comment by d33tah** [comment]
@ZuluPro excuse me, I didn't understand. Could you rephrase?

**Comment by ZuluPro** [comment]
@d33tah I think this issue has no place be as it should be a pull request to libcloud.

**Comment by d33tah** [comment]
@ZuluPro You're right that it's not an actual issue with the code - I just used this as a means for communicating with the developers about what could give them more exposure. I agree that ultimately there should be an attempt at creating a PR to libcloud if applicable.

**Comment by arubalucalasagni** [comment]
This is the right place to make this kind of requests, this permit us to know what are the needs of our Cloud users, naturally the lib cloud module for Aruba won't be made in pyArubaCloud. I have left this thread opened to remember us this demand. If you want contact us directly you can email us here cloudsdk@staff.aruba.it.

**Comment by vsilent** [comment]
As a software deployment startup we are interested in integrating aruba provider library into libcloud too.

**Comment by gbrault** [comment]
anything ready now?

**Comment by zakamarek** [comment]
Apache libCloud supports the biggest players on cloud market.  The idea to extend Apache libCloud with Aruba is really interesting and in fact it makes Aruba offer easy available for all libCloud users.
